selfdrive/car/hyundai/values.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, CP):
 class CAR:
   # Hyundai
   ELANTRA = "HYUNDAI ELANTRA 2017"
+  ELANTRA_21 = "HYUNDAI ELANTRA 2021"
```

Comment:
```suggestion
  ELANTRA_2021 = "HYUNDAI ELANTRA 2021"
```

Let's stay consistent with the Ionic naming.

Reply:
Got it, fixed.